docs/modules/release-notes/pages/5-4-0.adoc

```diff
@@ -0,0 +1,197 @@
+= 5.4.0 Release Notes
+
+== New Features
+
+* **xref:???.adoc[???]**: ???
+
+== Breaking Changes
+
+* Renamed the service port for Hazelcast clusters deployed in Kubernetes environments as `hazelcast`.
+Previously, the name was `hazelcast-service-port` causing the member auto-discovery (for embedded deployments) to fail.
+https://github.com/hazelcast/hazelcast/pull/24834[#24834], https://github.com/hazelcast/hazelcast/issues/24705[#24705], https://github.com/hazelcast/hazelcast/issues/24688[#24688]
+* The connector for Elasticsearch 6 does not exist anymore since the Elastichsearch 6 module has been removed from the Hazelcast distributions.
+https://github.com/hazelcast/hazelcast/pull/24734[#24734]
+* Fixed an issue where Hazelcast was not fast-failing when a member is started with a blank public address. This has been fixed by introducing a configuration validation
+which may break the existing invalid configurations.
+https://github.com/hazelcast/hazelcast/pull/24729[#24729]
+
```

Comment:
The method names to be used in the security interceptor are changed. These changes are in the RN as mentioned at [this](https://github.com/hazelcast/hazelcast/pull/25020) PR, commented as RN.